react/src/hooks/useNotification.ts

```diff
@@ -2,7 +2,7 @@ import { ToastPositionWithLogical, useMediaQuery, useToast } from "@chakra-ui/re
 import { useState, useEffect, useCallback } from "react";
 import { INotificationProps } from "./hooks";
 
-export const useNotification = () => {
+export const useNotification = (toastName?: string) => {
```

Comment:
@serchtul even if we switch the notification to the input field, I would definitely let this stay in. Quite a nice solution to check if a notification is active.